release-docker.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Prerequisites for releasing:
+
+# Logged in on Docker hub (docker login)
+
+# fail this script immediately if any command fails with a non-zero exit code
+set -eu
+
+function docker_compose {
+    if [[ "$1" == "up" ]]; then
+        docker-compose -f docker/docker-compose-tests.yml up --abort-on-container-exit
+    elif [[ "$1" == "down" ]]; then
+        docker-compose -f docker/docker-compose-tests.yml down -v
+    fi
+}
+
+function test_docker_image {
```

Comment:
Could we delegate this testing maybe to `integration-test.sh`, e.g. by calling it with a parameter? I think it would be ok to assume that this workflow is only called by a Makefile target.